packages/devextreme/js/ui/gantt/ui.gantt.treelist.nodes_state.js

```diff
@@ -33,6 +33,7 @@ export class GanttTreeListNodesState {
         this._resetHash();
     }
     applyNodes(nodes, rootValue) {
+        nodes = nodes.filter(node => node);
```

Comment:
Doubtful place for this code. Maybe discuss this with the Gantt team?

Reply:
Yup, I agree with Alyar

Also, let's don't override the function arguments
For cases like this, it's better to initialize a separate local variable :)